src/KM_Music.pas

```diff
@@ -98,7 +100,7 @@ constructor TMusicLib.Create(aVolume: Single);
   end;
   {$ENDIF}
 
-  UpdateMusicVolume(aVolume);
+  UpdateMusicVolume(aVolume); // Must stay here, sets audio volume to fade to.
```

Comment:
Old comment that I forgot to take out, wanted to set the volume to 0 there, but that would make no sense as it would overwrite the MusicGain value, that would be a no-go, plus it is useless to change something that has nothing to do with the issue.